src/packagedcode/utils.py

```diff
@@ -146,3 +147,22 @@ def build_description(summary, description):
             description = '\n'.join([summary , description])
 
     return description
+
+def combine_expressions(expressions, licensing=Licensing()):
+    """
+    Return a combined license expression string with AND, given a list of
+    license expressions.
+
+    For example:
+    >>> a = 'mit'
```

Comment:
Can you add a test for what happens when expressions is None, an empty list or a string or a string with a single character?

Reply:
Yep, I have added the tests on  the pypi branch, https://github.com/nexB/scancode-toolkit/commit/801bb5d92525fbc036e8fdab829ee41724f57f65